Jenkinsfile_updatecli

```diff
@@ -1,68 +1,47 @@
 def cronExpr = env.BRANCH_IS_PRIMARY ? 'H/30 * * * *' : ''
 
-pipeline {
-  agent {
-    label 'jnlp-linux-arm64'
-  }
-
-  options {
-    buildDiscarder(logRotator(numToKeepStr: '10'))
-    timeout(time: 30, unit: 'MINUTES')
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '10')),
     disableConcurrentBuilds()
-  }
-
-  triggers {
-    cron (cronExpr)
-  }
+])
+if (cronExpr) {
+    properties([pipelineTriggers([cron(cronExpr)])])
+}
```

Comment:
```suggestion
    properties([pipelineTriggers([cron(cronExpr)])]),
])
```

=> no conditional here, as the "condition" (e.g. "is it the principal branch or not) is already expressed on line 1 in the ternary form to set the value of the variable `cronExpr`.

Rationale: IF you are in a PR, then you want Jenkins to explicitly disable the cron trigger, by passing it the empty string. Otherwise it will inherit from the pipeline of the main branch.